ports/stm32/adc.c

```diff
@@ -673,7 +725,11 @@ float adc_read_core_vref(ADC_HandleTypeDef *adcHandle) {
     // update the reference correction factor
     adc_refcor = ((float)(*VREFIN_CAL)) / ((float)raw_value);
 
+    #if defined(STM32H7)
+    return raw_value * ADC_SCALE;
```

Comment:
Can you explain why this is different for the H7?

Reply:
No actually I can't, but using the old equation (using VREFIN_CAL) returns a very wrong reading, using raw_value * ADC_SCALE returns 1.193v (~1.2v) maybe it works like a normal channel now.

Reply:
In that case it's best not to make any changes here, just leave it returning a "wrong" value.  It can be fixed when a proper fix is known.